### PR TITLE
add example showing how to push to official NuGet

### PR DESCRIPTION
**Pull request by vslee**
## Summary
On the donet nuget push command, there's currently only an example showing how to push to a custom push source. This adds an example showing how to push to the official NuGet server.

### CODE REVIEW

File: docs/core/tools/dotnet-nuget-push.md [hunk 1]
```diff
@@ -92,7 +92,13 @@ The `dotnet nuget push` command pushes a package to the server and publishes it.
   dotnet nuget push foo.nupkg -k 4003d786-cc37-4004-bfdf-c4f3e8ef9b3a
   ```
 
-* Push *foo.nupkg* to the custom push source `https://customsource`, specifying an API key:
+* Push *foo.nupkg* to the offical NuGet server, specifying an API key:
+
+  ```dotnetcli
+  dotnet nuget push foo.nupkg -k 4003d786-cc37-4004-bfdf-c4f3e8ef9b3a -s https://api.nuget.org/v3/index.json
```
Comment (Youssef1313) [inline docs/core/tools/dotnet-nuget-push.md]: Is that your real API Key ? It's not safe to include it, I think.

```suggestion
  dotnet nuget push foo.nupkg -k YOUR_API_KEY -s https://api.nuget.org/v3/index.json
```

**Comment by vslee** [comment]
The key in the example is not my real API key. It is just the same made up key as in one of the other existing examples.

**Comment by Youssef1313** [comment]
> 
> 
> The key in the example is not my real API key. It is just the same made up key as in one of the other existing examples.

Yes I saw it in the existing example just after I wrote my comment, then I deleted it, but GitHub was already sent it by email I think 😄

**Comment by vslee** [comment]
> Yes I saw it in the existing example just after I wrote my comment, then I deleted it, but GitHub was already sent it by email I think 😄

Ah I see. Yes, I saw the email and went onto GH to reply, but was confused as to why I couldn't find your comment anywhere...

File: docs/core/tools/dotnet-nuget-push.md [hunk 2]
```diff
@@ -92,7 +92,13 @@ The `dotnet nuget push` command pushes a package to the server and publishes it.
   dotnet nuget push foo.nupkg -k 4003d786-cc37-4004-bfdf-c4f3e8ef9b3a
   ```
 
-* Push *foo.nupkg* to the custom push source `https://customsource`, specifying an API key:
+* Push *foo.nupkg* to the offical NuGet server, specifying an API key:
```
Comment (mairaw) [inline docs/core/tools/dotnet-nuget-push.md]:
```suggestion
* Push *foo.nupkg* to the official NuGet server, specifying an API key:
```

**Review by mairaw — COMMENTED** [review]
Thanks for your PR @vslee. @karann-msft please review.

**Review by karann-msft — APPROVED** [review]
🚢

**Review by mairaw — APPROVED** [review]

**Comment by mairaw** [comment]
Thank you @vslee. Your contribution will be live within a day or so.

**Pull request merged**